ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Update.scala

```diff
@@ -87,14 +87,14 @@ object Update {
     *   The ledger-provided timestamp at which the packages were uploaded.
     *
     */
-  final case class PublicPackagesUploaded(
-      archives: List[DamlLf.Archive],
+  final case class PublicPackageUploaded(
```

Comment:
Please fix the docstring above to adjust plurals to singulars.

Reply:
Done